packages/text-bitmap/src/utils/drawGlyph.ts

```diff
@@ -0,0 +1,90 @@
+import { generateFillStyle } from './generateFillStyle';
+import { hex2rgb, string2hex } from '@pixi/utils';
+import type { TextMetrics, TextStyle } from '@pixi/text';
+
+// TODO: Prevent code duplication b/w drawGlyph & Text#updateText
```

Comment:
oh yes! this is great idea!
we could create a cache that is mapped by style :D
love the idea of storing standard fonts as B/W and then tinting them! 👍